### PR TITLE
fix(app marketplace): prevent app marketplace image reload

### PR DESCRIPTION
**Pull request by oyo**
## Description

Prevent image reload on any user interaction on the app marketplace page

## Why

On user interaction on the app marketplace page all images are reloaded. This creates a very ugly flickering and results in a bad user experience. With this fix the images are cached and not reloaded every time.

## Issue

ref. Jira CPLP-3190

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Comment by jjeroch** [comment]
adding Nidhi since the issue should help on a couple of defects related to Nidhi her items

File: src/components/pages/AppMarketplace/components/AppListSection/index.tsx [hunk 1]
```diff
@@ -147,49 +94,17 @@ export default function AppListSection() {
...
-        <Typography
-          sx={{ fontFamily: 'LibreFranklin-Light' }}
-          variant="h3"
-          className="section-title"
-        >
-          {t('content.appstore.appOverviewSection.title')}
-        </Typography>
+        groupKey={control.group}
+      />
+    ) : (
```
Comment (manojava-gk) [inline src/components/pages/AppMarketplace/components/AppListSection/index.tsx:99]: Is this else case necessary?
why do not we use 
```
{data && (
<Component />
)}
```

**Review by manojava-gk — COMMENTED** [review]
small finding

File: src/components/pages/AppMarketplace/components/AppListSection/index.tsx (hunk 1 above)
Comment (oyo) [reply src/components/pages/AppMarketplace/components/AppListSection/index.tsx:99]: @manojava-gk a React component always needs to return a JSX element so on the top level a 
`data && <....>`
would result in an undefined return value. So we return the empty JSX item here.

**Review by Phil91 — APPROVED** [review]

**Pull request merged**